packages/concerto-core/lib/decoratormanager.js

```diff
@@ -77,11 +77,11 @@ enum MapElement {
 }
 
 /**
- * Applies a decorator to a given target
+ * Applies decorators to a given target
  */
 concept Command {
     o CommandTarget target
-    o Decorator decorator
+    o Decorator[] decorators
```

Comment:
This is a breaking change to the DCS metamodel. How can you make this change in a backwards-compatible way?